src/CargoBuild/README.md

```diff
@@ -0,0 +1,91 @@
+### CargoBuild Sdk Prototype
+
+### Project Setup
+To use this sdk you will need the following:
+
+1) in your global.json
+```json
+ "msbuild-sdks": {
+    ...,
+    "Microsoft.Build.CargoBuild": "1.0.270-gf406f8eaa0"
+  },
+```
+
+2) foreach rust project a .cargosproj project file at the same level as your cargo.toml file. The project file should import the CargoBuild sdk.
+```xml
```

Comment:
nit: In this XML blob I see 4 space indenting and the one below it is 2 space indenting.  I think XML should be 2 space indenting personally.